src/osd/osd_types.cc

```diff
@@ -248,7 +248,8 @@ void request_redirect_t::encode(bufferlist& bl) const
   ENCODE_START(1, 1, bl);
   encode(redirect_locator, bl);
   encode(redirect_object, bl);
-  encode(osd_instructions, bl);
+  bufferlist legacy_osd_inst;
+  encode(legacy_osd_inst, bl);
```

Comment:
even better is to ```encode((uint32_t)0, bl)```

Reply:
Cool, just the size of empty `bufferlist`. Thanks!